samples/bluetooth/peripheral_hids_mouse/overlay-nrf_rpc.conf

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
What do you think about naming it just nrf_rpc.conf?

Reply:
I think overlay should stay. It is consistent with other samples. Naming it nrf_rpc.conf would imply that it's a separate config, so all the Kconfig options would need to be copied there.

Reply:
Ok then